### ISSUE TITLE
difficult to decode normal barcodes on Glass

### ISSUE THREAD

**Issue by axd1967** [BarcodeEye/BarcodeEye#10]
Can anyone confirm that Zxing has a somewhat poor performance on Glass when scanning normal size (1D) barcodes?

I suspect an issue in BarcodeEye, because there is the same degradation when trying almost the same app on a Galaxy S4 mini.

See also [1]

Update: [2] is hopeful, and might obsolete BarcodeEye.

[1] https://groups.google.com/forum/#!topic/zxing/P-QLWhlB_wU
[2] https://groups.google.com/d/msg/zxing/P-QLWhlB_wU/hxg8h-Pj-jAJ

**Comment by jromero** [comment]
I don't believe Zxing is the primary issue with decoding normal barcode. After troubleshooting it for quite a while I was able to find some bugs in the GDK which I detailed here (https://github.com/BarcodeEye/BarcodeEye/issues/6#issuecomment-35570035) that would ease the recognition. It might be improved internally in Zxing but that would be something the Zxing team would need to look into when getting further into Google Glass.

[2] As noted on the thread and https://github.com/BarcodeEye/BarcodeEye/pull/3 the direction is a bit different for BarcodeEye. Hopefully, once the project starts up again, it would be more targeted to the end user and their experience rather than a catch all data displaying mechanism (which I have nothing against and am grateful it exists).

**Comment by headsvk** [comment]
Has anyone found a solution to this? I can't scan any 1D barcodes on products, just zoomed barcodes from the google doc. I also tried to set scene mode to barcode and set decode hints to TRY_HARDER and decode formats to ONE_D_FORMATS.

**Comment by team8bb** [comment]
Yeah I'm running into the same problem. If they're blown up it works fine, but it's having a hard time reading off physical barcodes; zxing on a galaxy s4 is much better. Tried setting the focus mode to near in the AutoFocusManager, but that was crashing the app.

**Comment by Thustle** [comment]
Still got the same issue. Has anyone had any success yet?

**Comment by jromero** [comment]
The Glass team seems to be very slow in fixing issues. I would put pressure
or at least reminders by staring and commenting on the issues linked to in
issue #6.

**Comment by dabikuru** [comment]
Has anyone had any luck...?
